### DISCUSSION TITLE
%E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/1130901%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1

### DISCUSSION THREAD

**Discussion by giscus** [cit-hall-43/cit-hall-43.github.io#7] [Q&A]
# %E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/1130901%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1

神人調和，繁茂復活.

https://cit-hall-43.github.io/%E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/1130901%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1.html

**Comment by cit-hall-43** [comment]
基督徒的生活完全是受試煉的生活。神並沒有應許我們在地上凡事如意。

也許你要問：禧年不是一切應心，萬事如意嗎？

然而，我們要知道，一切應心，並不應在一切事上，乃是應在神身上。

換言之，你若沒有神，事事不應心；你若有了神，事事就應心。

是否應心不在環境，乃在有沒有神。一旦你有了神，萬事就非常應心；即使有些事不順利，卻仍是應心的。

禧年的生活就是我們在任何景況中都取用基督自己的生活；基督在我們裡面作主因、作中心，要帶領我們克服人生一切的困擾。

（文/職事文摘 第九卷第一期）

 #水深之處
